src/sidebar/components/SearchInput.js

```diff
@@ -53,31 +53,69 @@ export default function SearchInput({ alwaysExpanded, query, onSearch }) {
     setPrevQuery(query);
   }
 
+  const isExpanded = alwaysExpanded || query;
+
   return (
     <form
       action="#"
-      className="SearchInput__form"
+      className={classnames(
+        // Relative positioning allows the search input to expand without
+        // pushing other things in the top bar to the right when there is
+        // a long group name (input will slide "over" end of group name in menu)
+        'relative',
+        'flex items-center',
+        // Having a nearly opaque white background makes the collision with
+        // group names to the left a little less jarring. Full white on hover
+        // to fully remove the distraction.
+        'bg-white/90 hover:bg-white transition-colors'
+      )}
       name="searchForm"
       onSubmit={onSubmit}
     >
-      <input
+      <TextInput
         aria-label="Search"
-        className={classnames('SearchInput__input', {
-          'is-expanded': alwaysExpanded || query,
-        })}
+        classes={classnames(
+          // This element is ordered second in the flex layout (appears to the
+          // right of the search icon-button) but having it first in source
+          // ensures it is first in keyboard tab order
```

Comment:
This matches the previous styles. I do wonder if we should just remove the search button from the tab sequence when the search bar is open, since all that happens when you click on the search button is that it gives the input  field focus.

Reply:
I need to do a bit of experimentation. I have a concern that the only way to access the search input field by keyboard is to tab to the search button, which focuses the button, which causes the input field to expand and then one can tab into it subsequently. The whole setup is funky, depending on CSS focus-styling. Looking forward to refactoring this in the future. I'll noodle just a bit, though I'm hesitant to rock the delicate boat too much right now.

Reply:
>  I do wonder if we should just remove the search button from the tab sequence **when the search bar is open**

I just realized I misread your comment.